content/install-hook-userstyles.js

```diff
@@ -79,6 +80,7 @@
     const md5Url = getMeta('stylish-md5-url');
     if (md5Url && installedStyle.md5Url && installedStyle.originalMd5) {
       getResource(md5Url).then(md5 => {
+        currentMd5 = md5;
```

Comment:
Is it possible that `currentMd5` is `undefined` when `md5Url && installedStyle.md5Url && installedStyle.originalMd5` is not truthy?

Reply:
`currentMd5` now set outside of the condition.